docs/recipes/pug.md

```diff
@@ -125,7 +125,9 @@ Recompile Pug templates on each change and reload the browser after an HTML file
 
 ```diff
  gulp.task('serve', ['views', 'styles', 'scripts', 'fonts'], () => {
-   ...
+-   runSequence(['clean', 'wiredep'], ['styles', 'scripts', fonts'], () => {
++   runSequence(['clean', 'wiredep'], ['html', 'styles', 'scripts', fonts'], () => {
```

Comment:
Yes, but I can't figure out how to amend this commit since I edited it via github. Would you rather I close this pull request and open a new one?